lib/tasks/migrations/migrations.rake

```diff
@@ -39,10 +40,14 @@ namespace :migrations do
     start, stop = range(args)
 
     has_encountred_issue = 0
+    filtered_roles_names = Role::RESERVED_ROLE_NAMES - %w[admin user]
```

Comment:
Do you want to use `Role::RESERVED_ROLE_NAMES` as the source of truth for this,
Or we hardcode it avoiding having issues on custom deployments that might have altered the list of reserved role names,